test_src/sljitTest.c

```diff
@@ -11288,37 +11301,57 @@ static void test90(void)
 
 	sljit_emit_enter(compiler, 0, SLJIT_ARGS2(VOID, W, W), 5, 5, 0, 0, 2 * sizeof(sljit_sw));
 
-	sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R0, 0, SLJIT_IMM, WCONST(0xffffa1b2c3d4, 0xa1b2c3d4));
```

Comment:
This is not need to be changed

Reply:
OK, but then the "description" of the case should change, or even better a different case which specifically handles the "junk" in the upper word is ignored by this instruction should be added instead IMHO

Reply:
The instruction reverses the lower 4 bytes, and sign/unsign extend the result. The upper 4 byte is ignored, since its value is not specified. The value is usually the result of some computation, and the upper bits depend on that. There is a REV32 variant, which uses the result of a 32 bit computation, so the upper bits might be defined by the cpu.